src/spell-check-handler.js

```diff
@@ -133,6 +133,7 @@ export default class SpellCheckHandler {
 
     this.scheduler = scheduler;
     this.shouldAutoCorrect = true;
+    this._automaticallyIdentifiyLanguages = true;
```

Comment:
This is fine, that's the shadow variable behind `automaticallyIdentifiyLanguages` 👍